Content.Shared/_RMC14/Medical/Stasis/CMStasisBagSystem.cs

```diff
@@ -55,7 +55,14 @@ private void OnInStasisRemove(Entity<CMInStasisComponent> ent, ref ComponentRemo
     private void OnInStasisGetInfectedIncubationMultiplier(Entity<CMInStasisComponent> ent, ref GetInfectedIncubationMultiplierEvent args)
     {
         if (ent.Comp.Running)
-            args.Multiply(ent.Comp.IncubationMultiplier);
+        {
+            // less effective in late stages
+            var multiplier = ent.Comp.IncubationMultiplier;
+            if (TryComp<VictimInfectedComponent>(ent, out var infection) && infection.CurrentStage >= infection.FinalSymptomsStart)
+                multiplier += (multiplier / 3);
+
+            args.Multiply(multiplier);
+        }
```

Comment:
It's probably better to add the speedup to the embryo code instead of here.

Or maybe pass the stage into the GetInfectedIncubationMultiplierEvent instead of getting the component here.